DEVX_ROADMAP.md

```diff
@@ -1,51 +1,39 @@
 
 ## Big picture
-* The gateway component will be deprecated. Clients will now communicate directly with full nodes, and full nodes will act as quorum driver (i.e., accept transactions, form a certificate + certificate effects, return the effects to the client)
-* Child objects are getting a big revamp: they no longer need to be passed as explicit transaction inputs, and rich new collection types (e.g., large hash maps with dynamic lookup) will be possible. More details in the Sui Move section.
+* The gateway component will be deprecated soon and we'll give community sufficient heads up once the dates are finalized. With this deprecation, clients will now communicate directly with full nodes, and full nodes will act as quorum driver (i.e., accept transactions, form a certificate + certificate effects, return the effects to the client).
+* Child objects revamp in still WIP. The features eliminates the need to explicitly pass transaction inputs, and enables rich new collection types (e.g., large hash maps with dynamic lookup).
+* We are adding new pay transaction types for generic payments. See below for more details.
 
 ## JSON RPC
-* [Gateway deprecation] `sui_syncAccountState` will be deprecated and removed.
-* [Gateway deprecation] Gateway’s `sui_executeTransaction` method will be replaced by fullnode’s `sui_executeTransaction` method which has an extra parameter that allows users to choose whether a method should block or return immediately.
-* Pagination support for the following APIs.
-  * `get_objects_owned_by_address`
-  * `get_objects_owned_by_object`
-  * `get_recent_transactions`
-  * `get_transactions_by_input_object`
-  * `get_transactions_by_mutated_object`
-  * `get_transactions_by_move_function`
-  * `get_transactions_from_addr`
-  * `get_transactions_to_addr`
-  * `get_events_by_transaction`
-  * `get_events_by_transaction_module`
-  * `get_events_by_move_event_struct_name`
-  * `get_events_by_sender`
-  * `get_events_by_recipient`
-  * `get_events_by_object`
-  * `get_events_by_timerange`
-* A `sui_call` API that emulates transaction execution, but does not commit the transaction to the network, akin to eth_call in Ethereum. This is useful for gas cost prediction and transaction effect simulation.
-* `sui_eventSubscription` API that enables subscribing to all transaction events, including ones that don’t generate network activity ([issue](https://github.com/MystenLabs/sui/pull/4288))
+* [Gateway deprecation] Working on adding metrics, logging and thorough testing of fullnodes. Also, working on a migration guide to enable smooth transition.
```

Comment:
-I think we can omit metrics, logging,and testing because these are are not visible to developers.
- should add 'Typescript' in front of migration guide to be specific.